ki/README.md

```diff
@@ -32,3 +32,68 @@ However, this feature was removed (perhaps temporarily) because the design of th
 # Documentation
 
 [Hackage documentation](https://hackage.haskell.org/package/ki/docs/Ki.html)
+
+# Example: Happy Eyeballs
+
+The [Happy Eyeballs](https://en.wikipedia.org/wiki/Happy_Eyeballs) algorithm is a particularly common example used to
+demonstrate the advantages of structured concurrency, because it is simple to describe, but can surprisingly difficult
```

Comment:
Shouldn't this says `but can *be* suprisingly difficult` ?

Reply:
Yep!